ios/Shared/ApplicationConfiguration.swift

```diff
@@ -37,4 +37,9 @@ enum ApplicationConfiguration {
 
     /// Maximum number of devices per account.
     static let maxAllowedDevices = 5
+
+    // swiftlint disable:force_cast
+    static let sameIPv4 = IPv4Address("10.127.255.254")!
+    static let sameIPv6 = IPv6Address("fc00:bbbb:bbbb:bb01:ffff:ffff:ffff:ffff")!
```

Comment:
Since the app in an ideal world should not need to know these IPs, I would add a comment, maybe a `// TODO` style comment, explaining that this is just temporary for aiding with debugging during migration to the Same IP system. And that it can be removed once the migration is done.

To help avoid it becoming arcane code living forever for no reason :)